themes/pelican-bootstrap3/static/css/style.css

```diff
@@ -130,9 +130,10 @@ body {
     margin-right: 3px;
 }
 
-a, a:hover {
+/*Make the links blue*/
```

Comment:
@elyezer @omaciel I can remove it, but I kept for a reason. As we are using a common theme, if we try to update it in future we will have the diff for this line and a comment explaining why it is commented out.